src/Models/Nodes.php

```
@@ -0,0 +1,39 @@
+ * @extends AbstractIndexedCollection<Node>
+ */
+final class Nodes extends AbstractIndexedCollection implements NodesInterface
+{
+    protected static string $itemType = Node::class;
+
+    protected static ?CollectionSchemaInterface $schema = null;
+
+    /**
+     * @return null|NodeInterface
+     */
+    public function current(): ?NodeInterface
```

PHPStan (2.1.15)

Return type (OpenFGA\Models\NodeInterface|null) of method OpenFGA\Models\Nodes::current() should be covariant with return type (OpenFGA\Models\Node|null) of method OpenFGA\Models\AbstractIndexedCollection<OpenFGA\Models\Node>::current()


(method.childReturnType)